roles/elastic-stack/ansible-elasticsearch/tasks/xpack_security.yml

```diff
@@ -149,6 +149,7 @@
   copy:
     src: "{{ item }}"
     dest: "{{ node_certs_destination }}/"
+    mode: 0444
   with_items:
```

Comment:
Read all permissions should not be assigned to a .key afaik

Reply:
Hello, @jm404, this perm is now correctly set to `0440` on https://github.com/wazuh/wazuh-ansible/pull/382/commits/4b9fb53549acd8a0fd2712ce59953827a3125f05